<?xml version="1.0" encoding="UTF-8"?>
<!-- Generated by org.testng.reporters.JUnitReportReporter -->
<testsuite hostname="Vinayak" failures="1" tests="5" name="remotePC.qa.testcases.LoginPageRPCTest" time="55.210" errors="0" timestamp="2022-04-28T21:27:57 IST" skipped="0">
  <testcase classname="remotePC.qa.testcases.LoginPageRPCTest" name="Invalidlogin" time="20.850"/>
  <system-out/>
  <testcase classname="remotePC.qa.testcases.LoginPageRPCTest" name="SignupRedirection" time="5.255"/>
  <system-out/>
  <testcase classname="remotePC.qa.testcases.LoginPageRPCTest" name="navigateToForgotPassword" time="11.035"/>
  <system-out/>
  <testcase classname="remotePC.qa.testcases.LoginPageRPCTest" name="ValidateLoginPage" time="2.444"/>
  <system-out/>
  <testcase classname="remotePC.qa.testcases.LoginPageRPCTest" name="loginToConsumerAccount" time="15.626">
    <failure message="Failed to receive title.Login failed expected [RemotePC™ - my account information] but found [RemotePC™ - Computers]" type="java.lang.AssertionError">
      <![CDATA[java.lang.AssertionError: Failed to receive title.Login failed expected [RemotePC™ - my account information] but found [RemotePC™ - Computers]
at org.testng.Assert.fail(Assert.java:99)
at org.testng.Assert.failNotEquals(Assert.java:1037)
at org.testng.Assert.assertEqualsImpl(Assert.java:140)
at org.testng.Assert.assertEquals(Assert.java:122)
at org.testng.Assert.assertEquals(Assert.java:629)
at remotePC.qa.testcases.LoginPageRPCTest.loginToConsumerAccount(LoginPageRPCTest.java:97)
at java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
at java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:78)
at java.base/jdk.internal.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
at java.base/java.lang.reflect.Method.invoke(Method.java:567)
at org.testng.internal.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:133)
at org.testng.internal.TestInvoker.invokeMethod(TestInvoker.java:598)
at org.testng.internal.TestInvoker.invokeTestMethod(TestInvoker.java:173)
at org.testng.internal.MethodRunner.runInSequence(MethodRunner.java:46)
at org.testng.internal.TestInvoker$MethodInvocationAgent.invoke(TestInvoker.java:824)
at org.testng.internal.TestInvoker.invokeTestMethods(TestInvoker.java:146)
at org.testng.internal.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:146)
at org.testng.internal.TestMethodWorker.run(TestMethodWorker.java:128)
at java.base/java.util.ArrayList.forEach(ArrayList.java:1511)
at org.testng.TestRunner.privateRun(TestRunner.java:794)
at org.testng.TestRunner.run(TestRunner.java:596)
at org.testng.SuiteRunner.runTest(SuiteRunner.java:377)
at org.testng.SuiteRunner.runSequentially(SuiteRunner.java:371)
at org.testng.SuiteRunner.privateRun(SuiteRunner.java:332)
at org.testng.SuiteRunner.run(SuiteRunner.java:276)
at org.testng.SuiteRunnerWorker.runSuite(SuiteRunnerWorker.java:53)
at org.testng.SuiteRunnerWorker.run(SuiteRunnerWorker.java:96)
at org.testng.TestNG.runSuitesSequentially(TestNG.java:1212)
at org.testng.TestNG.runSuitesLocally(TestNG.java:1134)
at org.testng.TestNG.runSuites(TestNG.java:1063)
at org.testng.TestNG.run(TestNG.java:1031)
at org.testng.remote.AbstractRemoteTestNG.run(AbstractRemoteTestNG.java:115)
at org.testng.remote.RemoteTestNG.initAndRun(RemoteTestNG.java:251)
at org.testng.remote.RemoteTestNG.main(RemoteTestNG.java:77)
]]>
    </failure>
  </testcase> <!-- loginToConsumerAccount -->
  <system-out/>
</testsuite> <!-- remotePC.qa.testcases.LoginPageRPCTest -->
